cs/unittest/TestSearch.cs

```diff
@@ -145,10 +145,10 @@ void RunSearchPredict(Multiline multiline)
                     driver.ForEachMultiline(rawvw.Learn);
 
                     rawvw.EndOfPass();
-                    driver.Reset();
+                    //driver.Reset(); -- Throws if search is called on out of bound learner
```

Comment:
we dont need this anymore with your patch right?

Reply:
I think this fails, I can check again

Reply:
@lokitoth do you have any theory on why this change would break this test?